portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesWithXSS.testcase

```diff
@@ -144,4 +144,41 @@ definition {
 		}
 	}
 
+	@description = "This is for LPS-132705."
```

Comment:
This is a test for LPS-132705. The user could select a category with XSS name in Account Settings.